test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -1707,3 +1707,48 @@ TEST_F(TestContentFilterSubscriptionUse, content_filter_reset) {
     }
   }
 }
+
+TEST_F(
+  CLASSNAME(
+    TestContentFilterSubscriptionUse,
+    RMW_IMPLEMENTATION), create_two_content_filters_with_same_topic_name_and_destroy) {
```

Comment:
```suggestion
TEST_F(TestContentFilterSubscriptionUse, create_two_filters_with_same_topic_name_and_destroy) {
```

Reply:
Srry, I forwarded the fix from `humble` and did not noticed that test definition changed